Sdtrig.tex

```diff
@@ -86,13 +86,19 @@ \section{Actions}
 \hline
 5 & Reserved for use by the trace specification. \\
 \hline
-8 -- 9 & Signal the firing of the trigger to other blocks within the hart (e.g. as countable events to hpmcounters).  Use external debug trigger output 0 or 1 (respectively). \\
+8 -- 9 & Send a signal to TM external trigger output 0 or 1 (respectively).  the
+firing of the trigger to blocks. \\
 \hline
 other & Reserved for future use. \\
 \hline
 \end{tabular}
 \end{table}
 
+\begin{commentary}
+    Actions 8 and 9 are intended to increment custom event counters, but these
+    signals could also be brought out to pins for use by external logic.
```

Comment:
Maybe this is just me but I think that `s/out to pins/to outputs/` would be better.

Reply:
That's more general. Thanks.